### ISSUE TITLE
error when creating HF_Datasets instance

### ISSUE THREAD

**Issue by deep-diver** [richarddwang/hugdatafast#2]
the following code snippets
```python
dls = HF_Datasets(tokenized_datasets, cols=['text_idxs', 'label'], hf_toker=hf_tokenizer)
```

generates an error below
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-9-c30dd56e3ec8> in <module>()
      1 tokenized_datasets = datasets.map(simple_tokenize_func({'sentence':'text_idxs'}, hf_tokenizer))
----> 2 dls = HF_Datasets(tokenized_datasets, cols=['text_idxs', 'label'], hf_toker=hf_tokenizer)

2 frames
/usr/local/lib/python3.6/dist-packages/fastcore/utils.py in store_attr(names, self, but, **attrs)
     95     args,varargs,keyw,locs = inspect.getargvalues(fr)
     96     if self is None: self = locs[args[0]]
---> 97     if not hasattr(self, '__stored_args__'): self.__stored_args__ = {}
     98     if attrs: return _store_attr(self, **attrs)
     99 

AttributeError: 'str' object has no attribute '__stored_args__'
```

seems like fastcore has got upgraded?

**Comment by deep-diver** [comment]
looks like fastcore has been upgraded recently.
if you are using `store_attr` function, please make sure to update its usage.

AFAIK, the old version is like

```python
class Foo():
    def __init__(self, a, b):
        store_attr(self, 'a, b')
```

and the new version is like

```python
class Foo():
    def __init__(self, a, b):
        store_attr()
```

**Comment by richarddwang** [comment]
Thanks for the report.
I will fix it tomorrow.

**Comment by richarddwang** [comment]
Hi, this is now fixed in the latest version (hugdatafast==0.4.4).
Try `pipi install -U hugdatafast`.
If it sill happen, please reopen the issue.
